.devcontainer/devcontainer.json

```
@@ -2,24 +2,23 @@
 {
     "name": "Turing Data Safe Haven - Pulumi",
```

```suggestion
    "name": "Turing Data Safe Haven",
```
do you still want the `- Pulumi` here?